docs/source/user-guide/tuning.md

```diff
@@ -105,8 +105,15 @@ then any shuffle operations that cannot be supported in this mode will fall back
 
 ## Metrics
 
-Comet metrics are not directly comparable to Spark metrics in some cases.
+Some Comet metrics are not directly comparable to Spark metrics in some cases.
```

Comment:
This might require specifics otherwise its too unclear

Reply:
Thanks. I have updated this.